src/simulation/m_global_parameters.fpp

```diff
@@ -615,22 +616,27 @@ contains
         #:endfor
 
         do j = 1, num_probes_max
+            mono(j)%pulse = dflt_int
+            mono(j)%support = dflt_int
             do i = 1, 3
                 mono(j)%loc(i) = dflt_real
             end do
             mono(j)%mag = dflt_real
             mono(j)%length = dflt_real
-            mono(j)%delay = dflt_real
+            mono(j)%wavelength = dflt_real
+            mono(j)%frequency = dflt_real
+            mono(j)%gauss_sigma_dist = dflt_real
+            mono(j)%gauss_sigma_time = dflt_real
+            mono(j)%npulse = dflt_real
             mono(j)%dir = dflt_real
-            mono(j)%npulse = dflt_int
-            mono(j)%pulse = dflt_int
-            mono(j)%support = dflt_int
+            mono(j)%delay = dflt_real
```

Comment:
I think we agreed that we could default this to zero. Otherwise, we have to change all the existing input `case.py` files to have `delay -> 0`, right?

Reply:
Regarding this, I've set the default delay to 0 for sine wave, but not for Gaussian pulse:

```
            if (f_is_default(mono(i)%delay)) then ! m_checker guarantees mono(i)%delay is set for pulse = 2 (Gaussian)
                delay(i) = 0d0 ! Defaults to zero for sine and square waves
            else
                delay(i) = mono(i)%delay
            end if
```

This is because it doesn't make sense for Gaussian pulse's t0 to default to zero - it'll become a half-Gaussian pulse. The original implementation used t0=3.5*sigt, where sigt is the sigma_t for the Gaussian bell curve. This makes sense but is very arbitrary, so I removed the default and enforced that the user must input a value. 

By only defaulting the sine wave's delay to zero, the existing input case.py files will still work. (I don't think any existing tests or examples uses Gaussian pulse)

Reply:
hmm. my understanding of how i implemented the delay feature was that it doesn't "turn on" the source term until a certain time is reached... of course a Gaussian wave is infinitely "wide," so one is forced to create a Gaussian with some fixed standard deviation/width. does that make any sense?

Reply:
You are right that a Gaussian pulse is infinitely wide and we need to cut it off somewhere. 3.5 is a very reasonable choice, but the user should understand and have control over the parameter. A Gaussian pulse in an acoustic simulation is most commonly used to find the impulse response, in which case an insufficient delay can cause a large discontinuity that shows up as oscillations in the frequency domain. The threshold for "sufficient delay" is determined by the specific simulation setup and the required accuracy of the computed transfer function, so I think it is crucial for anyone using it to know exactly where the pulse is cut off, and a default value shouldn't be used.

Reply:
seems reasonable to me